### PR TITLE
login and create user ui forms

### PR DESCRIPTION
**Pull request by biomassives**
the mui form  TextField loses its title when in focus, im trying to fix that

### CODE REVIEW

File: src/frontend/main/src/routes.jsx [hunk 1]
```diff
@@ -16,6 +20,22 @@ const routes = createBrowserRouter(
...
+                    path: "/login",
+                    element: <Login />,
+                },
+                {
+                    path: "/signup",
+                    element: <Create />,
+                },
+                {
+                    path: "/recoveraccess",
+                    element: <Forgot />,
```
Comment (emi420) [inline src/frontend/main/src/routes.jsx:37]: I would be more explicit with the component's name here, for clarity and consistency.

Create -> SignUp
Forgot -> RecoverAccess

**Comment by robsavoye** [comment]
There were some changes for the .env variables yesterday. Will those changes break this ?

File: src/frontend/main/src/styles/login.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+button {
+    border-radius: 5px;
+    background: #54af29;
```
Comment (emi420) [inline src/frontend/main/src/styles/login.css:3]: I think styled components and MUI theme are being used for this project:

https://github.com/hotosm/fmtm/blob/main/src/frontend/main/src/utilities/CustomizedMenus.tsx#L7

File: src/frontend/main/src/views/Forgot.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+import enviroment from "../environment";
+import Axios  from "axios";
+
+
+
+const Forgot = () => {
+    // const state:any = useSelector<any>(state=>state.project.projectData)
+    // console.log('state main :',state)
+
+    const { type } = windowDimention();
```
Comment (emi420) [inline src/frontend/main/src/views/Forgot.tsx:22]: Window's dimension is not being used.

File: src/frontend/main/src/views/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+*/
+
+
+
+
+const Login = () => {
+    // const state:any = useSelector<any>(state=>state.project.projectData)
+    // console.log('state main :',state)
+
+    const { type } = windowDimention();
```
Comment (emi420) [inline src/frontend/main/src/views/Login.tsx:27]: Window's dimension is not being used.

File: src/frontend/main/src/views/Tabbed.tsx [hunk 5]
```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import TabsUnstyled from '@mui/base/TabsUnstyled';
+import TabsListUnstyled from '@mui/base/TabsListUnstyled';
+import TabPanelUnstyled from '@mui/base/TabPanelUnstyled';
+import { buttonUnstyledClasses } from '@mui/base/ButtonUnstyled';
+import TabUnstyled, { tabUnstyledClasses } from '@mui/base/TabUnstyled';
+
+const blue = {
```
Comment (emi420) [inline src/frontend/main/src/views/Tabbed.tsx:9]: These colors maybe belong to the MUI Theme instead of the view.

**Review by emi420 — COMMENTED** [review]
Hey @biomassives ! I added a few comments only, for naming consistency and styles/theming.

File: src/frontend/main/src/routes.jsx (hunk 1 above)
Comment (biomassives) [reply src/frontend/main/src/routes.jsx:37]: thank you, Emilio  for the useful feedback..  i had noticed this.  it seems best is when Create -> path; /create   ,,

File: src/frontend/main/src/styles/login.css (hunk 2 above)
Comment (biomassives) [reply src/frontend/main/src/styles/login.css:3]: updated for PR

File: src/frontend/main/src/views/Tabbed.tsx (hunk 5 above)
Comment (biomassives) [reply src/frontend/main/src/views/Tabbed.tsx:9]: new  dev view for use in profile page im thinking..   
good \<o>

**Pull request merged**